github/downloader_test.go

```diff
@@ -151,3 +164,107 @@ func TestOnlineOrganizationDownload(t *testing.T) {
 	}
 
 }
+
+func loadReqResp(filepath string, reqResp map[string]string) error {
+	// Open a RO file
+	decodeFile, err := os.Open(filepath)
+	if err != nil {
+		return err
+	}
+	defer decodeFile.Close()
+	// Create a decoder
+	decoder := gob.NewDecoder(decodeFile)
+	// Decode
+	decoder.Decode(&reqResp)
+	return nil
+}
+
+// RoundTripFunc .
+type RoundTripFunc func(req *http.Request) *http.Response
+
+// RoundTrip .
+func (f RoundTripFunc) RoundTrip(req *http.Request) (*http.Response, error) {
+	return f(req), nil
+}
+
+func TestOfflineOrganizationDownload(t *testing.T) {
+	var (
+		reqResp map[string]string = make(map[string]string)
+		err     error
+	)
+	// Load the recording
+	err = loadReqResp("../testdata/organization_src-d_2019-10-08.gob", reqResp)
+	if err != nil {
+		t.Errorf("Failed to read the offline recordings:%s", err)
+	}
+
+	// Setup the downloader with RoundTrip functionality.
+	// Not using the NewStdoutDownloader initialization because it overides the transport
```

Comment:
hm... it means retry isn't get tested. Maybe it is ok. Just a note for other reviewers.

Reply:
Similar to https://github.com/src-d/metadata-retrieval/pull/29/files#r332986229 I will add an issue to complete it after this PR of happy paths.

Reply:
Opened #40